packages/react/src/components/layout/CarouselView.tsx

```diff
@@ -1,4 +1,4 @@
-import type { TrackReferenceOrPlaceholder } from '@livekit/components-core';
+import { TrackReferenceOrPlaceholder, getScrollBarWidth } from '@livekit/components-core';
```

Comment:
I think `TrackReferencePlaceholder` should still be a type import

Reply:
Well spotted! Prettiere and Autoimport can mess up the imports.